### ISSUE TITLE
[BUG] No module named cli

### ISSUE THREAD

**Issue by Nishith-Savla** [apoclyps/reviews#41]
Whenever I try to run it in gives the following traceback

## Excepted behaviour:
Dashboard would open

## Actual behaviour:
```
Traceback (most recent call last):
  File "c:\users\admin\appdata\local\programs\python\python39\lib\runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "c:\users\admin\appdata\local\programs\python\python39\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python39\Scripts\reviews.exe\__main__.py", line 4, in <module>
ModuleNotFoundError: No module named 'cli'
```

**Comment by apoclyps** [comment]
@Nishith-Savla thanks for raising the issue. 

I don't usually depend on windows outside of WSL2 so I missed this; I've updated the code base in #43 to resolve the issues with building and installing `reviews` on Windows.  You can use the following steps to check if your issue is resolved.

### Steps to reproduce

```bash
git clone https://github.com/apoclyps/code-review-manager
cd code-review-manager

pip install -e .

set REPOSITORY_CONFIGURATION="apoclyps/my-dev-space"
set GITHUB_TOKEN=your-token

reviews dashboard

# ctrl + pause break to exit
```

Although I managed to get the dashboard running on Windows, the experience was no on par with running it on a UNIX system. For this reason, I've dropped official support for windows (until such time as the experience improves). I would be happy to review and accept any PRs to add support for Windows (if this is something you would like).

**Comment by Nishith-Savla** [comment]
Thanks, that worked. Actually, that wasn't an OS-specific problem because I had tried it on WSL2 (Kali Linux) as well before starting the issue and it didn't work there as well. It may be better to have a look on it as well.

**Comment by apoclyps** [comment]
A transitive dependency changed which needed to be pinned inhttps://github.com/apoclyps/code-review-manager/pull/43/files#diff-4d7c51b1efe9043e44439a949dfd92e5827321b34082903477fd04876edb7552R8 which why it was breaking cross-platform.

As for window specific issues, the path to an SQLite database (used for a local cache - still being developed) was failing in https://github.com/apoclyps/code-review-manager/pull/43/files#diff-54086b89c0a6d38c0d89a17324d9075c8612308e14b0188014dc75be7ee1dbbbL17 because the paths didn't exist on the host machine. These are no longer loaded by default.

As for running it on windows, I encountered slow rendering/flickering when the screen was being updated - the user experience was quite poor - I'd be interesting to know if you encounter this issue.

**Comment by Nishith-Savla** [comment]
Not yet will update if I see one

**Comment by waynew** [comment]
FWIW  that's because `cli.py` is outside the source directory. A much nicer approach would be to move cli to say, `reviews/__main__.py` or even `reviews/cli.py`. 

If you move the source into a `src` folder, it will expose several issues along those lines.

For now, I'm able to work around this by cloning the repo, doing

```
python -m venv env
python -m pip install -e .
ln -s /path/to/repo/env/bin/reviews ~/.local/bin/reviews
```

You can see the existing buggy behavior by simply not installing in editable mode (i.e. 

```
python -m pip install .
cd /tmp/
/path/to/repo/env/bin/reviews -h
```
)

**Comment by apoclyps** [comment]
Thanks, @waynew for the feedback. I'll work on reproducing this locally and reopen this issue in the meantime.

**Comment by apoclyps** [comment]
@waynew I've started work on fixing this in https://github.com/apoclyps/reviews/pull/128 - I've opened it as a draft - feel free to provide any feedback you feel is relevant. All opinions and thoughts are welcome.

**Comment by apoclyps** [comment]
I've refactored the module and how it's run to ensure it's installable without `--editable` alongside testing other methods for running the module/cli in https://github.com/apoclyps/reviews/pull/128

**Comment by apoclyps** [comment]
This has been fixed and will be released in version 0.1.8: https://github.com/apoclyps/reviews/pull/133
